bootstrap/bootstrap/src/bootstrap.rs

```diff
@@ -26,15 +26,19 @@ fn bootstrap_zos(cfg: &config::Config) -> Result<()> {
     let flist = match &cfg.runmode {
         RunMode::Prod => match &cfg.version {
             Version::V3 => "zos:production-3:latest.flist",
+            _ => bail!("unsupported version in old style"),
```

Comment:
it's safe to completely remove the old style from code since ALL environments have been updated to new style. We needed to keep the old style for a while because bootstrap is shared between all environments (since it's part of base image) because some envs used new style and others didn't.

Anyway, you can assume that old style does no longer exist